torch/_functorch/aot_autograd.py

```diff
@@ -499,6 +499,15 @@ def __post_init__(self):
         # sanity assert to make sure we don't leak memory
         assert is_fake(self.original_subclass)
 
+
+def _is_mutation_recorded_in_the_graph(input_info: InputAliasInfo):
+    return input_info.mutates_data and not input_info.mutates_metadata and not m.requires_grad
+
+
+def _is_mutation_reflected_in_the_runtime(input_info: InputAliasInfo):
```

Comment:
Hmm, why is this function not a simple opposite of `_is_mutation_recorded_in_the_graph` (why can't we just kill this function and do `not _is_mutation_recorded_in_the_graph()`?

Reply:
Oh this should have been gone. I never use this